generators/server/templates/src/test/java/package/cucumber/stepdefs/_StepDefs.java

```diff
@@ -2,13 +2,15 @@
 
 import <%=packageName%>.<%= mainClass %>;
 
-import org.springframework.boot.test.SpringApplicationContextLoader;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.web.WebAppConfiguration;
 import org.springframework.test.web.servlet.ResultActions;
 
+import org.springframework.boot.test.context.SpringBootTest;
+
 @WebAppConfiguration
-@ContextConfiguration(classes = <%= mainClass %>.class, loader = SpringApplicationContextLoader.class)
+@SpringBootTest
+@ContextConfiguration(classes = <%= mainClass %>.class)
```

Comment:
@tan9 I tried this, or at least if I understand your suggestion correctly, but I get NPEs from `mvn test` if I omit the `@ContextConfiguration` annotation. You're right that we could move the `classes` argument to `@SpringBootTest` as per [the docs](http://docs.spring.io/spring-boot/docs/1.5.1.RELEASE/api/org/springframework/boot/test/context/SpringBootTest.html#classes--) but would it actually matter if we have to have both annotations anyway?